charts/ccpay-callback-function/values.preview.template.yaml

```
@@ -5,19 +5,20 @@ function:
     APPINSIGHTS_INSTRUMENTATIONKEY: 124e39c3-6dad-49ef-aede-d2cddac865c2
     SERVICE_CALLBACK_SUBSCRIPTION: pr-subscription-name
   secrets:
-    SERVICE_CALLBACK_BUS_CONNECTION:
-      secretRef: servicebus-secret-topic-{{ .Release.Name }}-servicebus-servicecallbacktopic
+    SB_ACCESS_KEY:
+      secretRef: fees-pay-sb-preview
+      key: primaryKey
+    SB_CONNECTION_STRING:
+      secretRef: fees-pay-sb-preview
       key: connectionString
-    SERVICE_CALLBACK_SUBSCRIPTION:
-      secretRef: servicebus-secret-topic-{{ .Release.Name }}-servicebus-servicecallbacktopic
-      key: subscriptionName
 servicebus:
   resourceGroup: fees-pay-rg
+  sbNamespace: "fees-pay-sb-preview"
```

according to https://github.com/hmcts/cnp-flux-config/blob/fd9533d0f13f60d07d062ccee00a5f3186f319b2/apps/fees-pay/preview/aso/fees-pay-servicebus.yaml#L4
```suggestion
  sbNamespace: "fees-pay-servicebus-preview"
```

---

DONE